eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -631,6 +657,30 @@ function ($pathString) use ($query, $handler) {
         $query->prepare()->execute();
     }
 
+    private function buildHiddenSubtreeQuery(string $selectExpr): QueryBuilder
+    {
+        $query = $this->connection->createQueryBuilder();
+        $expr = $query->expr();
+        $query
+            ->select($selectExpr)
+            ->from('ezcontentobject_tree', 't')
+            ->leftJoin('t', 'ezcontentobject', 'c', 't.contentobject_id = c.id')
+            ->where(
+                $expr->orX(
```

Comment:
orX is deprecated
```suggestion
                $expr->or(
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/3142/commits/66ee6a667394b4bb962d330baef104e4e07ef042